TodoList/sources/TodoList.Api/App_Start/FormatConfig.cs

```diff
@@ -1,16 +1,11 @@
-﻿using System.Linq;
-using System.Net.Http.Formatting;
-using System.Web.Http;
+﻿using System.Web.Http;
 using Newtonsoft.Json.Serialization;
 
 namespace TodoList.Api
 {
     internal static class FormatConfig
     {
-        internal static void Register(HttpConfiguration config)
-        {
-            var jsonFormatter = config.Formatters.OfType<JsonMediaTypeFormatter>().First();
-            jsonFormatter.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
-        }
+        internal static void Register(HttpConfiguration config) 
+            => config.Formatters.JsonFormatter.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
```

Comment:
Within the next task, consider wrapping accessor line by line (before each `.`) to increase readability.